packages/generator/src/utils/fetch-latest-version-for.ts

```diff
@@ -26,6 +26,8 @@ export const fetchLatestVersionsFor = async <T extends Record<string, string>>(
 
   return {
     isFallback: fallbackUsed,
-    value: Object.fromEntries(updated),
+    value: updated.reduce((object, [key, value]) => {
```

Comment:
can we use a more descriptive variable for the accumulator, something like `acc` or `result`? `object` is just a bit of a loaded term.